panacea/query_client.go

```diff
@@ -395,3 +396,39 @@ func (q QueryClient) GetOracleParamsPublicKey() (*btcec.PublicKey, error) {
 
 	return btcec.ParsePubKey(pubKeyBz, btcec.S256())
 }
+
+func (q QueryClient) GetDeal(dealID uint64) (*datadealtypes.Deal, error) {
```

Comment:
If possible, please add a test code. or if impossible don't added a test code :)
Maybe you can edited `/panacea/test/data/panacea-core.init.sh`

Reply:
To make a `deal` in panacea, we need to put the `buyer_address` in the json file containing the deal info (e.g. create_deal.json) and generate a transaction with the json file as a input.
I tried modifying the `/panacea/test/data/panacea-core.init.sh`, but it doesn't work right away. I will try again and upload it through another PR.

Reply:
OK. It's not important now :)